app/controllers/kudos_controller.rb

```diff
@@ -43,26 +35,23 @@ def update
     end
   end
 
-  # DELETE /kudos/1
   def destroy
     @kudo.destroy
     redirect_to kudos_url, notice: 'Kudo was successfully destroyed.'
   end
 
   def correct_employee
```

Comment:
this method has misleading name - in general the name of the method should explain what it is doing - so here I am expecting, that I will receive a `correct_employee`, but instead I am getting a `kudo`
Consider naming it something like `verify_kudo_giver` or somethign like that - also do you really need to return `@kudo`?

Reply:
Fixed.